pwa-devdocs/src/tutorials/intercept-a-target/add-new-environment-variable/index.md

```diff
@@ -0,0 +1,259 @@
+The intercept function appends a new definition to the [core environment variable definitions][], which allows frontend code access to the `IMAGE_PLACEHOLDER_SERVICE_URL` environment variable.
+
+## Test on a local instance
+
+Install this package in a local storefront project to use the PlaceholderImage component.
+
+```sh
+yarn add --dev link:/path/to/your/project
+```
+
+This adds a `devDependencies` entry to your storefront project's `package.json` that looks like the following:
+
+```diff
+    "@storybook/react": "~5.2.6",
++   "PlaceholderImage": "link:/path/to/your/project",
+    "apollo-cache-persist": "~0.1.1",
+```
+
+### Create environment variable entry
+
+In your project's `.env` file, create an entry for `IMAGE_PLACEHOLDER_SERVICE_URL`.
+
+```text
+IMAGE_PLACEHOLDER_SERVICE_URL=http://www.loremflickr.com/${w}/${h}
+```
+
+The PlaceholderImage component uses this value to compute the source when it renders the image.
+
+### Create a demo page
+
+Create a demo page component to render the ImagePlaceholder component.
+
+```sh
+mkdir -p src/components/PlaceholderImageDemo && touch src/component/PlaceholderImageDemo/placeholderImageDemo.js
+```
+
+Inside `placeholderImageDemo.js`, add the following content:
+
+```jsx
+import React from 'react'
+import PlaceholderImage from 'PlaceholderImage'
+
+const PlaceholderImageDemo = props => {
+
+    return <PlaceholderImage width={200} height={300} />
+}
+
+export default PlaceholderImageDemo
+```
+
+### Add a static route
+
+If you used the project scaffolding tool in PWA Studio 8.0.0, your project will have a `local-intercept.js` file.
+If you do not have this file, use the same earlier steps to [create and register the intercept file][].
+
+Inside your storefront's intercept file, add the following content to add a new static route for your demo page:
+
+```js
+function localIntercept(targets) {
+  targets.of("@magento/venia-ui").routes.tap((routes) => {
+    routes.push({
+      name: "Placeholder Image demo page",
+      pattern: "/placeholder-image-demo",
+      exact: true,
+      path: require.resolve(
+        "./src/components/PlaceholderImageDemo/placeholderImageDemo.js"
+      )
+    });
+    return routes;
+  });
+}
+
+module.exports = localIntercept
+```
+
+### Check out the page
+
+Now, when you start your project, you can navigate to `/placeholder-image-demo` and see the PlaceholderImage component in action.
+
+Click to see the full solution for this tutorial:
+
+[![Edit environment-variable-tutorial](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/environment-variable-tutorial-9z0rb?fontsize=14&hidenavigation=1&module=%2Fextensions%2FPlaceholderImage%2Fsrc%2Fcomponents%2FPlaceholderImage%2FplaceholderImage.js&theme=dark)
+
+### A note about CodeSandbox
+
+CodeSandbox uses an iframe to display a preview of the site.
```

Comment:
I think it would be reasonable to remove the `@magento/upward-security-headers` package in your sandbox template, and you can get rid of this section.

Reply:
that worked. thanks!